NvAPIWrapper/Native/Display/Structures/MasteringDisplayData.cs

```diff
@@ -0,0 +1,117 @@
+﻿using System.Runtime.InteropServices;
+
+namespace NvAPIWrapper.Native.Display.Structures
+{
+    [StructLayout(LayoutKind.Sequential, Pack = 8)]
+    public struct MasteringDisplayData
+    {
+        internal readonly ushort _DisplayPrimaryX0;
+        internal readonly ushort _DisplayPrimaryY0;
+        internal readonly ushort _DisplayPrimaryX1;
+        internal readonly ushort _DisplayPrimaryY1;
+        internal readonly ushort _DisplayPrimaryX2;
+        internal readonly ushort _DisplayPrimaryY2;
+        internal readonly ushort _DisplayWhitePointX;
+        internal readonly ushort _DisplayWhitePointY;
+        internal readonly ushort _MaxDisplayMasteringLuminance;
+        internal readonly ushort _MinDisplayMasteringLuminance;
+        internal readonly ushort _MaxContentLightLevel;
+        internal readonly ushort _MaxFrameAverageLightLevel;
+
+        /// <summary>
+        ///    coordinate of color primary 0 (e.g. Red) of mastering display ([0x0000-0xC350] = [0.0 - 1.0])
+        ///</summary>
+        public ushort DisplayPrimaryX0
+        {
+            get => _DisplayPrimaryX0;
+        }
+
+        /// <summary>
+        ///     coordinate of color primary 0 (e.g. Red) of mastering display ([0x0000-0xC350] = [0.0 - 1.0])
+        ///</summary>
+        public ushort DisplayPrimaryY0
+        {
+            get => _DisplayPrimaryY0;
+        }
+
+        /// <summary>
+        ///    coordinate of color primary 1 (e.g. Green) of mastering display ([0x0000-0xC350] = [0.0 - 1.0])
+        ///</summary>
+        public ushort DisplayPrimaryX1
+        {
+            get => _DisplayPrimaryX1;
```

Comment:
These values can be converted to `float` or `double` by normalization. It feels more natural like that for the library's user.

Reply:
done

Reply:
Well, it would be also great to normalize these, like so:

```C#
        public float DisplayPrimaryY0
        {	       
            get => (float)_DisplayPrimaryY0 / 0xC350;
        }
```

This will always result in a number between 0-1. Which is easier for the user to work with instead of forcing them to know the upper limit.

Reply:
Ah, cool!
I'm clearly new to this kind of thing :P

done!

Reply:
Thanks Matt. Some of the properties have a range of [1-0xFFFF] which also converted to `float` normalized as [0-0xC350]. Please keep these as `ushort` as [1-0xFFFF] is already the range of valid values for `ushort` anyway and the user can't get confused with them.

Sorry that I am not helping you in this since I am currently working on custom resolutions and my work directory is already a mess.

Reply:
`MaxDisplayMasteringLuminance` as an example.